contents/handbook/growth/sales/yc-onboarding.md

```diff
@@ -69,7 +69,7 @@ Read more about running demos on our [dedicated page](/handbook/growth/sales/dem
 
 A super cool way of doing demos for websites and web apps is using the PostHog bookmarklet. This way, you can set up funnels, charts, and other views for the user (or, ideally, have them set it up themselves) by leveraging their *own data*. If done on their instance, some of these views might even be ready for them to start analyzing user data once "real events" start coming in.
 
-To learn more about the bookmarklet, check out our [Snippet Installation](/docs/integrate/client/snippet-installation#get-started-with-no-code) page.
+To learn more about the bookmarklet, check out our [Snippet Installation](/docs/integrate) page.
```

Comment:
The bookmarklet will be no more shortly...

Reply:
Got it, I say it's easiest to just go through once we do nuke it and remove all references then :+1: